servers/rendering/renderer_rd/cluster_builder_rd.h

```diff
@@ -49,7 +49,7 @@ class ClusterBuilderSharedDataRD {
 	RID cone_vertex_array;
 	RID cone_index_buffer;
 	RID cone_index_array;
-	float cone_overfit = 0.0; //because an cone mesh is not a perfect sphere, we need to enlarge it to cover the actual cone area
+	float cone_overfit = 0.0; //because an cone mesh is not a perfect cone, we need to enlarge it to cover the actual cone area
```

Comment:
Style nitpick. This should use our standard formatting of starting with a space and a capital and ending with a period

Reply:
I took the liberty to fix the formatting (+ sometimes grammar/clarity) of all comments in `cluster_builder_rd.h/cpp`. That might be a bit out of scope, but I thought better now than never :)

Reply:
Wonderful!